src/shared/utils.ts

```diff
@@ -212,6 +212,22 @@ export function escapeHTML(
     return output;
 }
 
+/**
+ * Returns a string containing the label and readable keyboard shortcut for button tooltips
+ * @param label The type to toggle to
+ * @param mapping? Corresponding command mapping (keyboard shortcut)
+ */
+export function getShortcut(mapping: string): string {
+    if (mapping.indexOf("Mod-") !== 0) {
+        return mapping;
+    }
+
+    return mapping.replace(
+        "Mod-",
+        /Mac|iP(hone|[oa]d)/.test(navigator.platform) ? "Cmd-" : "Ctrl-"
+    );
```

Comment:
optional: This code is totally, 1000% fine as-is, but I'm feeling pedantic and in the mood for micro-optimizations today:

```suggestion
    if (!mapping.startsWith("Mod-")) {
        return mapping;
    }

    return (/Mac|iP(hone|[oa]d)/.test(navigator.platform) ? "Cmd" : "Ctrl") + mapping.slice(3);
```